smarthome.go

```diff
@@ -84,55 +137,43 @@ func (s *SmartHome) handleExecuteIntent(agentUserId string, req proto.ExecReques
 		}
 	}
 	responseBody := proto.ExecResponse{}
-	// for each command
+	responses := make(chan proto.CommandResponse)
+	resCount := 0
```

Comment:
Maybe just `var resCount int`, could be a little bit clearer and would produce the same results ?